phpcs.xml.dist

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<ruleset>
+	<!-- Files or directories to check -->
+	<file>.</file>
+
+	<exclude-pattern>*/node_modules/*</exclude-pattern>
+   <exclude-pattern>*/wordpress/*</exclude-pattern>
+	<exclude-pattern>*/vendor/*</exclude-pattern>
+   <exclude-pattern>*/resources/*</exclude-pattern>
+	<exclude-pattern>*/dist/*</exclude-pattern>
+	<exclude-pattern>*/tests/*</exclude-pattern>
+
+	<!-- Path to strip from the front of file paths inside reports (displays shorter paths) -->
+	<arg name="basepath" value="." />
+
+	<!-- Set a minimum PHP version for PHPCompatibility -->
+	<config name="testVersion" value="7.4-" />
```

Comment:
Should we use PHP 8.1 here maybe, since we will require Timber 2.2.0, which requires PHP 8.1 as well?

Reply:
Yes, I will update the tests matrix as well to 8.1